code/API_definitions/application-profiles.yaml

```
@@ -33,59 +59,59 @@ paths:
     post:
       security:
         - openId:
-            - connectivity_insights:policy:write
+            - application-profile:write
```

As `api-name` is _application-profiles_
```suggestion
            - application-profiles:write
```